pymc4/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .model import Model
+from .random_variable import *
```

Comment:
for `import *` statements, the file should define an `__all__` variable.  This prevents the namespace from getting full of unwanted stuff (right now `from pymc4 import ed` would work, because `random_variable.py` imports edward).